llvm/lib/Transforms/Utils/LoopUtils.cpp

```diff
@@ -1192,6 +1192,19 @@ Value *llvm::createSimpleTargetReduction(IRBuilderBase &Builder, Value *Src,
   }
 }
 
+Value *llvm::createSimpleTargetReduction(VectorBuilder &VBuilder, Value *Src,
+                                         const RecurrenceDescriptor &Desc) {
+  RecurKind Kind = Desc.getRecurrenceKind();
+  assert(!RecurrenceDescriptor::isAnyOfRecurrenceKind(Kind) &&
```

Comment:
Is this ensured somewhere? Do we have tests with any-of reductions?

Reply:
99081ed139ead25e3a756abdc0f76cc4a3da0398
Added test case for IAnyOf and FAnyOf.